### DISCUSSION TITLE
Yazi is amazing

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [sxyazi/yazi#1341] [1. Q&A]
I absolutely love yazi. I've just recently spent 2 days configuring `lf` to find out 1 hour after I finished that there is yazi, which works even better. I migrated my whole config with no regrets.

I think you may like this chart, looks sick:
<img width="899" alt="image" src="https://github.com/user-attachments/assets/3b4cffe3-3c58-4cba-b04e-8718fb76f35e">

https://star-history.com/#sxyazi/yazi&gokcehan/lf&ranger/ranger&Date

**Comment by xfzv** [comment]
Agreed! I also migrated from `lf` after using it for about a year. I'm not going back.
